### ISSUE TITLE
Parameter guideline 'wcag' seems not to filter on tests for wcag

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#123]
When we give quail the paramter 'guideline'='wcag', all tests seems to run, also not wcag2, for example 'all links should have title attribute'
